source/help/messaging/executing-commands.rst

```diff
@@ -41,11 +41,16 @@ The following slash commands are available on all Mattermost installations:
     "/settings", "Open the Account Settings dialog", "/settings"
     "/shortcuts", "Display a list of keyboard shortcuts", "/shortcuts"
     "/shrug *{message}*", "Add ``¯\_(ツ)_/¯`` to your message", "/shrug oh well"
+    "/status *{emoji_name}* *{descriptive status_message}* *", Set a custom status that includes an optional emoji and a descriptive status message, "/status sick Feeling unwell and taking time off to recover" 
+    "/status clear * ", Clear the current status, "/status clear" 
 
 Begin by typing `/` and a list of slash command options appears above the text input box. The autocomplete suggestions help by providing a format example in black text and a short description of the slash command in grey text.
 
 .. image:: ../../images/slashCommandsAutocomplete.PNG
 
+.. note::
+  Custom status slash commands marked with an asterisk (*) in the table above will be available in the Mattermost Mobile App in a future release.
```

Comment:
```suggestion
  Custom status slash commands marked with an asterisk [*] in the table above will be available in the Mattermost Mobile App in a future release.
```

Reply:
This seems to work better but not sure it looks good.

Reply:
@justinegeffen Removed the need to include asterisks in the table. See above for workaround.